README.md

```diff
@@ -742,6 +742,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [NuPIC](https://github.com/numenta/nupic) - Numenta Platform for Intelligent Computing.
 * [Pylearn2](https://github.com/lisa-lab/pylearn2) - A Machine Learning library based on [Theano](https://github.com/Theano/Theano).
 * [scikit-learn](http://scikit-learn.org/) - The most popular Python library for Machine Learning.
+* [surprise](http://surpriselib.com) - A scikit for building and analyzing recommender systems.
```

Comment:
This is weird, it works all fine for me (it's hosted on GitHub pages).

What's on the site is nothing but the README of the project though, so you can check it up [here](https://github.com/NicolasHug/Surprise).